modules/@apostrophecms/doc-type/index.js

```diff
@@ -253,6 +276,20 @@ module.exports = {
 
   methods(self) {
     return {
+      getRelatedDocsIds(req, doc) {
+        const relatedDocsIds = [];
+        const handlers = {
+          relationshipHandler: (doc, field) => {
```

Comment:
The more I look at it, the more I think the word "Handler" is redundant here. It would be better if it were just "relationship", "object" etc.

Reply:
Found another way, by merging the provided handlers with the default ones